apps/ukvi-complaints/lib/utils.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const { v4: uuidv4 } = require('uuid');
+const Validator = require('jsonschema').Validator;
+const { Producer } = require('sqs-producer');
+const config = require('../../../config');
+const decsSchema = require('../schema/decs.json');
+
+const validateAgainstSchema = (complaintData) => {
```

Comment:
Change to `data` rather than `complaintData` to be more generic

Reply:
Done in commit bec1e53